gen/calc.py

```diff
@@ -1009,7 +1010,8 @@ def calculate_fault_domain_detect_contents(fault_domain_detect_filename):
         'mesos_master_work_dir': '/var/lib/dcos/mesos/master',
         'mesos_agent_work_dir': '/var/lib/mesos/slave',
         'fault_domain_detect_filename': 'genconf/fault-domain-detect',
-        'fault_domain_detect_contents': calculate_fault_domain_detect_contents
+        'fault_domain_detect_contents': calculate_fault_domain_detect_contents,
+        'license_key_contents': '',
```

Comment:
Will this allow us to override this in config.yaml *instead* of having a license.txt file under `genconf`?

Reply:
cc @branden

Reply:
I explained how enterprise handles the license key param here: https://github.com/mesosphere/dcos-enterprise/pull/1982#discussion_r160215989

Reply:
Thanks @branden !